scripts/check_markdown_frontmatter.py

```
@@ -0,0 +1,133 @@
+import subprocess
+import sys
+from pathlib import Path
+
+
+REQUIRED_KEYS = ("title", "when_to_read", "summary", "ontology_relations")
+RELATION_KEYS = ("relation", "target", "note")
+
+
+def tracked_markdown_files() -> list[Path]:
+    result = subprocess.run(
+        ["git", "ls-files", "*.md"],
```

Ruff (0.15.11)

[error] Starting a process with a partial executable path

(S607)